submit.yml.erb

```diff
@@ -5,6 +5,6 @@ batch_connect:
 
 script:
     native:
-        - "-l host=k095,ngpus=1,ncpus=6,mem=40gb,walltime=12:00:00"
-    walltime: 43200
+        - "-l host=k095,ngpus=1,ncpus=8,mem=125gb,walltime=08:00:00"
+    walltime: 28800
```

Comment:
Residual from when I was testing things. Reverted the walltime.
Might be worth keeping the cpu/mem just because that's how much is available per GPU anyway - can discuss in the meeting.